aircraft.py

```diff
@@ -0,0 +1,22 @@
+class AircraftHandler:
+    def __init__(self, displayHandler):
+        self.g13 = displayHandler
+
+    def buttonHandleSpecificAC(self, buttonPressed):
```

Comment:
It might applicable only for FA18, so every jet needs its own method. Will test it later on F16

Reply:
it both `specelF16Handler` and `specelFA18Handler` implementation of `buttonHandleSpecificAC()` before refactoring looked exactly the same. That's why I move it to base class. If any newer plane needs other behavior, it can simply redefine method in it's class.

Reply:
Yup, looked exactly the same, because it was copy and paste. Anyway, you were right with refactoring, just my sloppy coding. I will change it later this week

Reply:
I've just check F-16 DCSBIOS and it looks, we need:

* **ICP_COM1_BTN**
* **ICP_COM2_BTN**

or something from  **"category": "UFC"**

Reply:
I guess for now i leave it like this due to F16 specifics - I might find for them different function. It felt natural to handle rotarty comm switches on Hornet with these buttons, but on Viper there's different way of manipulating comms. Anyway, I'll release it with yours changes, whole thing looks much better now